### PR TITLE
Fix more broken links

### PR DESCRIPTION
**Pull request by doublek**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

File: calico/about/about-network-policy.md
Comment (caseydavenport) [inline calico/about/about-network-policy.md:219]: Should this just be `https://docs.tigera.io/security/tiered-policy` so we get the latest release?

**Review by caseydavenport — APPROVED** [review]
@doublek one question but otherwise LGTM

**Comment by doublek** [comment]
Oops. I had the merge ready on this. I'll fix up and put up another PR.

**Pull request merged**
